templates/repo/issue/view_content/comments.tmpl

```diff
@@ -495,9 +495,22 @@
 		{{else if eq .Type 25}}
 			<div class="timeline-item event">
 				<span class="badge">{{svg "octicon-git-branch"}}</span>
-				{{template "shared/user/avatarlink" dict "user" .Poster}}
+				{{if .OriginalAuthor}}
+				{{else}}
```

Comment:
Why an empty block?

* If it needs to show something ... then show something
* If it doesn't need to show anything, it could be `{{if not ...}}`

Reply:
I copied this from elsewhere in this file and didn't want to break convention. I'll update it here.